definitions/variable/energy/energy-prices.yaml

```diff
@@ -0,0 +1,112 @@
+    weight: Primary Energy|Biomass
+
+- Price|Primary Energy|Coal:
+    description: Coal price at the primary level (i.e. the spot price at the global or
+      regional market)
+    unit: [EUR_2020/GJ, USD_2010/GJ]
+    weight: Primary Energy|Coal
+
+- Price|Primary Energy|Gas:
+    description: Natural gas price at the primary level (i.e. the spot price at
+      the global or regional market)
+    unit: [EUR_2020/GJ, USD_2010/GJ]
+    weight: Primary Energy|Gas
+
+- Price|Primary Energy|Oil:
+    description: Crude oil price at the primary level (i.e. the spot price at
+                the global or regional market)
+    unit: [EUR_2020/GJ, USD_2010/GJ]
+    weight: Primary Energy|Oil
+
+# Secondary energy prices:
+- Price|Secondary Energy|{Final Energy carrier}:
+    description: price at the secondary level of {Final Energy carrier}, i.e. for large scale
+                consumers (e.g. aluminum production, gas power plant)
+    notes: Prices should include the effect of carbon prices.
+    unit: [EUR_2020/GJ, USD_2010/GJ]
+    weight: Secondary Energy|{Final Energy carrier}
+
+# Final energy prices:
+- Price|Final Energy|Industry|{Final Energy carrier}:
+    description: price at the final level in the industrial sector of {Final Energy carrier}. 
+    notes: Prices should include the effect of carbon prices and all other taxes.
+    unit: [EUR_2020/GJ, USD_2010/GJ, EUR_2020/kWh]
+    weight: Final Energy|Industry|{Final Energy carrier}
+
+- Price|Final Energy|Transportation|{Final Energy carrier}:
+    description: price at the final level in the transport sector of {Final Energy carrier}. 
+    notes: Prices should include the effect of carbon prices and all other taxes.
+    unit: [EUR_2020/GJ, USD_2010/GJ, EUR_2020/kWh]
+    weight: Final Energy|Industry|{Final Energy carrier}
+
+- Price|Final Energy|Residential and Commercial|{Final Energy carrier}:
+    description: price at the final level in the residential and commercial sector of {Final Energy carrier}. 
+    notes: Prices should include the effect of carbon prices and all other taxes.
+    unit: [EUR_2020/GJ, USD_2010/GJ, EUR_2020/kWh]
+    weight: Final Energy|Industry|{Final Energy carrier}
+
+- Price|Final Energy|Residential|{Final Energy carrier}:
+    description: price at the final level in the residential sector of {Final Energy carrier}. 
+    notes: Prices should include the effect of carbon prices and all other taxes.
+    unit: [EUR_2020/GJ, USD_2010/GJ, EUR_2020/kWh]
+    weight: Final Energy|Industry|{Final Energy carrier}
+
+- Price|Final Energy|commercial|{Final Energy carrier}:
+    description: price at the final level in the commercial sector of {Final Energy carrier}. 
+    notes: Prices should include the effect of carbon prices and all other taxes.
+    unit: [EUR_2020/GJ, USD_2010/GJ, EUR_2020/kWh]
+    weight: Final Energy|Industry|{Final Energy carrier}
+
+# additional variables    
+- Price|Secondary Energy|Gases|Natural Gas:
```

Comment:
It would make sense to also add this as a tag-list in this PR...

Reply:
This variable was kept only due to backwards compatibility reasons in the db. 
After some clarification with Robert, my understanding is that the "Natural Gas" keyword should be replaced by "Gas" for consistence in the NAVIGATE template, and secondary energy prices should follow the structure of final energy carriers. 
Therefore, the variable "Price|Secondary Energy|Gases|Fossil" should already reflect the natural gas prices at secondary energy level as far as I understand, and natural gas specific variables would make more sense at the primary energy level.

Nevertheless, if both of you think otherwise (@danielhuppmann and @robertpietzcker ), I can add this additional split to the {Final Energy carrier} tag, although I think it is unecessary.

Reply:
To my understanding, having the SE-prices on the disaggregation level fossil/electricity/biomass is sufficient for most questions at hand, so adding the prices for all the subcomponents of fossil (Gas/Oil/Coal for the energy carriers gases, liquids, electricity, heat and hydrogen) will create many new variables without adding much insights at the moment. 

So I would go with Renato's proposal to not move that to the full tagging system.

I am not sure how much backwards compatibility is needed. In principle, I would be in favor of dropping these variables, as they simply don't fit the new system anymore.  If backwards compatibility is needed on the SE level, I would propose to just keep the three idiosyncratic variables "Price|Secondary Energy|Gases|Natural Gas", "Price|Secondary Energy|Solids|Coal" "Price|Secondary Energy|Liquids|Oil".
(although it pains me a bit to keep this one instance of "Natural Gas" when we decided to remove the "Natural" from all other variables and just stick to "Gas")

Reply:
From the scenario explorer point of view it is not necessary to keep this variable as these definitions only affect data during upload. So if there's no new data coming with this variable it would be fine to be removed.
For bookkeeping purposes we might want to keep it though but maybe expand the `notes` attribute to make it clear that this variable is no longer to be used.